samples/MATLAB/colorspace.m

```diff
@@ -0,0 +1,36 @@
+function obj = colorspace(obj,col)
```

Comment:
You mentioned you modified the file. Could you clarify how and why you modified it?
Was it [this file](https://github.com/DIPlib/diplib/blob/cf71858a91991fc2c8796e4f9a12f57643a24f0b/dipimage/%40dip_image/dip_image.m) initially? If it was, I think we should rather put it in `tests/fixtures/MATLAB/`.

Reply:
Yes, that's the file. It's kind of large, I thought you would not want such a large example just for the sake of one particular comment line. But I'd be happy changing that commit to include the whole file.

Reply:
Is this what you had in mind?

Reply:
Nop, you were right to change it. We avoid including large files because of memory issues (see #2117 for details).

We also try to only use original files to train the Bayesian classifier, so I think we should put this file in `tests/fixtures/MATLAB/`, where the Bayesian classifier won't use it.

Reply:
OK, I've put the file I originally submitted in the `tests/fixtures/MATLAB` directory.